tests/test_outputs/conftest.py

```diff
@@ -5,14 +5,18 @@
 from typing import Callable
```

Comment:
I'm not sure how conftest works: if it provides a fixture for the other tests, why do we still need to read in the config in test_intram_by_pg?

Reply:
Changed this to import the function